.github/ISSUE_TEMPLATE/questions-and-help.md

```diff
@@ -2,7 +2,7 @@
 name: Questions and Help
 about: Ask questions or request help
 title: ''
-labels: needs triage, question
+labels: ''
```

Comment:
Was removing `question` here intentional?

Reply:
yes, `Needs repro` tells people this issue needs attention and I think that's good enough.

Reply:
Ok. I thought maybe `question` was useful for clarifying the user's expectations. It makes me wonder if the question template is needed. Your call. I don't know anything about PTVS issues...